src/main/webapp/firebase-chat.js

```diff
@@ -0,0 +1,280 @@
+/* eslint-disable */
+const firebaseConfig = {
```

Comment:
Confirming here. Will some one with these details be able to write to the DB? Or are there any ACLs that restrict it?

Reply:
firestore has security rules we need to implement to restrict the kinds of requests made to our database documents, we will have to write that logic ourselves eventually